CMakeLists.txt

```diff
@@ -11,7 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-cmake_minimum_required(VERSION 3.11)
+cmake_minimum_required(VERSION 3.14)
+
+# the policy allows us to change options without caching
+cmake_policy(SET CMP0077 NEW)
```

Comment:
Can you educate me on how this is helpful ?

Reply:
Let me cite from the [docs](https://cmake.org/cmake/help/latest/policy/CMP0077.html) as they contain our use case: 
> For example, a project that embeds another project as a subdirectory may want to hard-code options of the subproject to build the way it needs.

Reply:
Thanks !